lib/BackbeatConsumer.js

```diff
@@ -161,6 +147,34 @@ class BackbeatConsumer extends EventEmitter {
         return this;
     }
 
+    _incrementMetrics(entry) {
+        const qEntry = QueueEntry.createFromKafkaEntry(entry);
+        if (!qEntry.error) {
```

Comment:
Looks like we should include a check that this an instance of `ObjectQueueEntry`, as it can sometimes be a `BucketQueueEntry` instance, in which case there is no method `getContentLength`.